app/uk/gov/hmrc/agentclientrelationships/services/CheckAndCopyRelationshipsService.scala

```diff
@@ -71,14 +75,21 @@ class CheckAndCopyRelationshipsService @Inject()(
   @Named("features.copy-relationship.mtd-vat") copyMtdVatRelationshipFlag: Boolean)
     extends Monitoring {
 
-  def checkForOldRelationshipAndCopy(arn: Arn, identifier: TaxIdentifier, eventualAgentUser: Future[AgentUser])(
+  def checkForOldRelationshipAndCopy(
+    arn: Arn,
+    identifier: TaxIdentifier,
+    eventualAgentUser: Future[Either[String, AgentUser]])(
     implicit ec: ExecutionContext,
     hc: HeaderCarrier,
     request: Request[Any],
     auditData: AuditData): Future[CheckAndCopyResult] = {
 
     def ifEnabled(copyRelationshipFlag: Boolean)(body: => Future[CheckAndCopyResult]): Future[CheckAndCopyResult] =
-      if (copyRelationshipFlag) body else returnValue(CopyRelationshipNotEnabled)
+      if (copyRelationshipFlag) {
+        body
+      } else {
+        returnValue(CopyRelationshipNotEnabled)
```

Comment:
this `returnValue` is just returning Future.successful,
the name is misleading, we can use simply like `toFuture(....)` and 
let's use the same method everywhere instead of `Future.successful`, but maybe in the upcoming PR